src/test/java/com/aws/greengrass/lifecyclemanager/LogManagerHelperTest.java

```diff
@@ -123,9 +131,122 @@ void GIVEN_mock_service_WHEN_getComponentLogger_THEN_logs_to_correct_log_file()
         assertEquals(0, ggLogFile.length());
     }
 
+    @Test
+    void GIVEN_mock_service_logger_WHEN_reconfigure_THEN_change_applied_correctly() throws IOException {
+        Path tempRootDir2 = tempRootDir.resolve("2");
+        Path tempRootDir3 = tempRootDir.resolve("3");
```

Comment:
`2` and `3` are too arbitrary... would prefer more meaningful names so that it could be identified when necessary.